openapi/components/schemas/Coupon/CouponRestrictions/minimum-item-quantity.yaml

```
@@ -0,0 +1,20 @@
+description: |
+  Minimum invoice item quantity restriction.
+  Specifying productIds will only apply the restriction to those invoice items.
+  If productIds is an empty array then it will apply to all invoice items.
```

```suggestion
  Restricts a coupon redemption to invoice items with the quantity greater or equal to specified.
```
I think it's a general description of a restriction and describing every parameter is not necessary. Better to describe parameter in place